guides/customizing-invoices.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc
+sublayout: guide
+title: Customizing Invoices
+---
+
+Invoices provided by Quay.io try to provide as much information as possible.
+However, it be necessary to include extra information for regulations such as Value Added Tax (VAT).
```

Comment:
Yar, it be necessary to include additional words here!

"However, it can be necessary...."